meta-resin-rocko/recipes-core/util-linux/util-linux_%.bbappend

```diff
@@ -0,0 +1,6 @@
+ALTERNATIVE_${PN} += "fallocate fsfreese nsenter unshare"
+ALTERNATIVE_LINK_NAME[fallocate] = "${bindir}/fallocate"
+ALTERNATIVE_LINK_NAME[fsfreese] = "${sbindir}/fsfreeze"
+ALTERNATIVE_LINK_NAME[nsenter] = "${bindir}/nsenter"
+ALTERNATIVE_LINK_NAME[unshare] = "${bindir}/unshare"
+
```

Comment:
Extra new line for all these util-linux bbappends.

Reply:
Removed